params/config.go

```diff
@@ -344,6 +344,12 @@ var NetworkNames = map[string]string{
 	HoleskyChainConfig.ChainID.String(): "holesky",
 }
 
+// Arbitrum
+// ArbOSInit defines some initialization values for ArbOS state.
+type ArbOSInit struct {
+	NativeTokenSupplyManagementEnabled bool `json:"nativeTokenSupplyManagementEnabled"`
```

Comment:
I think this still needs to be a timestamp. Because, for chains where the feature is currently disabled, we want there to be a 7-day delay before it can be enabled. So, we need to implement it as a time after which the mint and burn precompiles can be called.

Reply:
Done.